backend/migrations/default/1629362271453_alter_table_public_Enrollment_alter_column_MotivationGrade/up.sql

```diff
@@ -0,0 +1 @@
+alter table "public"."Enrollment" rename column "MotivationGrade" to "MotivationRating";
```

Comment:
The name of the foreign key is still `Enrollment_MotivationGrade_fkey`. This was quite confusing!